docs/gpa.md

```diff
@@ -180,25 +179,30 @@ This requires the [Covered By](https://github.com/open-contracting-extensions/oc
         <td>the subject-matter of the procurement;</td>
         <td markdown=1>
 
-* 
+* Enter it in `tender/description`
+* If `tender/description` is not empty, enter it after the existing content
 </td>
       </tr>
       <tr>
         <td>VII:3(b)</td>
         <td>the final date for the submission of tenders or, where applicable, any final date for the submission of requests for participation in the procurement or for inclusion on a multi-use list; and</td>
         <td markdown=1>
 
-* 
+* Enter *the final date for the submission of tenders* or, if disclosed, *any final date for the submission of  requests for participation in the procurement or for inclusion on a multi-use list*, in `tender/tenderPeriod/endDate`
 </td>
       </tr>
       <tr>
         <td>VII:3(c)</td>
         <td>the address from which documents relating to the procurement may be requested.</td>
         <td markdown=1>
 
-* 
+* If the address has been entered in section VII:2(a), discard
```

Comment:
If a Notice of Intended Procurement has already been published as OCDS data, I don't think the publisher needs to publish anything relating to a Summary Notice, since the Notice of Intended Procurement already has all that information.

Do we just have a paragraph explaining this, instead of mapping each article?

(Also, use "Article" instead of "section", like in the GPA text https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm)

Reply:
From IVV-3, the "subject-matter of the procurement" is the only bit that is not specifically present in the Notice of Intended Procurement. Do "melt" it in "description of the procurement" from VII-2(b)?

Reply:
Yes, I think that's fine. I don't imagine it can refer to anything not already in the other notice.

Reply:
Done